Dockerfile

```diff
@@ -16,17 +16,17 @@ FROM alpine:3.8
 
 ARG REPO=/go/src/github.com/hyperledger/burrow
 
-ENV USER monax
-ENV MONAX_PATH /home/$USER/.monax
+ENV USER burrow
+ENV BURROW_PATH /home/$USER
 RUN addgroup -g 101 -S $USER && adduser -S -D -u 1000 $USER $USER
-WORKDIR $MONAX_PATH
+WORKDIR $ BURROW_PATH
```

Comment:
this won't work. can we just remove this file from the commit and use mine? it makes the same changes and adds a bit better hygeine to the image.